pkg/validation/internal/operatorhub.go

```diff
@@ -17,51 +17,71 @@ import (
 	interfaces "github.com/operator-framework/api/pkg/validation/interfaces"
 )
 
+const minKubeVersionWarnMessage = "csv.Spec.minKubeVersion is not informed. It is recommended you provide this information. " +
+	"Otherwise, it would mean that your operator project can be distributed and installed in any cluster version " +
+	"available, which is not necessarily the case for all projects."
+const apiDeprecationMsg = "apiextensions/v1beta1 for CRDs and admissionregistration.k8s.io/v1beta1 for Webhooks was " +
```

Comment:
Split this into one message per GK.

Reply:
I think it is no longer required since we check the crds and raise the info msg.
So, I am closing. However, please feel free to re-open if you see that we should change that.

Reply:
In that case, remove the webhook bit from the message